src/Kuzzle.php

```diff
@@ -101,6 +105,12 @@ public function __construct($url, array $options = [])
             $this->requestHandler = new CurlRequest();
         }
 
+        if (array_key_exists('port', $options)) {
+            $this->port = $options['port'];
+        }
+
+
+        $this->url = 'http://' . $host . ':' . $this->port;
```

Comment:
Actually, adding SSL support involves more work than just using `https` in the URL: we also have to manage certificates, either to ignore them and/or (preferably "and") by allowing clients to use one.
I think that adding SSL support should be done in a separate PR.